server/controllers/DocumentControllers.js

```diff
@@ -0,0 +1,190 @@
+import pagination from '../helper/pagination';
+
+const Document = require('../models').Document;
+const User = require('../models').User;
+
+const DocumentControllers = {
+  create(req, res) {
+    const authorId = res.locals.decoded.UserId;
+    return Document.create({
+      title: req.body.title,
+      content: req.body.content,
+      UserId: authorId,
+      access: req.body.access
+    })
+    .then(document => res.status(201).send({
+      message: 'Document successfully created',
+      document,
+    })
+    )
+    .catch(error => res.status(400).send(error));
+  },
+
+  /**
+   *
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns the list of documents available
+   */
+  listDocuments(req, res) {
+    if (req.query) {
+      const limit = req.query.limit || 10;
+      const offset = req.query.offset || 0;
+      return Document.findAndCount({
+        limit,
+        offset,
+      })
+    .then(document => res.status(200).send({
+      pagination: {
+        row: document.rows,
+        paginationDetails: pagination(document.count, limit, offset)
+      }
+    }))
+    .catch(error => res.status(403).send(error));
+    }
+  },
+
+  /**
+   *
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns the requested document
+   */
+  getDocument(req, res) {
+    return User
+    .findById(res.locals.decoded.UserId, {
+      include: [{
+        model: Document,
+        as: 'documents',
+      }],
+      attributes: ['fullname', 'username', 'email']
+    })
+    .then((user) => {
+      if (!user) {
+        return res.status(404).send({
+          message: 'User Not Found',
+        });
+      }
+      return res.status(200).send({
+        message: `These are the documents created by ${res.locals.decoded.userFullname}`,
+        user,
+      });
+    })
+    .catch(error => res.status(400).send(error));
```

Comment:
Please do not send the errors directly back to the user, ensure to handle these sequelize errors.